tests/unit/design_document_tests.py

```diff
@@ -24,6 +24,8 @@
 import json
 import os
 import unittest
+from mock import mock
```

Comment:
I think your code will still work if you just use `import mock` here instead of `from mock import mock`.